server/routers/admin_dashboard.py

```
@@ -125,246 +166,195 @@ def get_recent_users(limit: int = Query(5, ge=1, le=100), current_admin: User =
-from datetime import datetime, timedelta
-from db_utils.db import SessionLocal, User, Disaster, Post
-from middleware.admin_auth import get_current_admin
 
-router = APIRouter()
 
-
-def get_db():
-    db = SessionLocal()
+@router.get("/logs/stats")
+async def get_log_stats(
+    db: Session = Depends(get_db),
```

Ruff (0.14.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)